corehq/apps/reports/standard/export.py

```diff
@@ -431,11 +439,38 @@ class FormExportReport(FormExportReportBase):
     @property
     def template_context(self):
         context = super(FormExportReport, self).template_context
-        export_id = self.request.GET.get('export_id')
+        # TODO - seems redundant, cleanup at some point
         context.update({
-            'export': SavedExportSchema.get(export_id),
-            'additional_params': 'export_id=%(export_id)s' % {
-                'export_id': export_id,
-            },
+            'export': self.exports[0],
+            'exports': self.exports,
+            "use_bulk": len(self.export_ids) > 1,
+            'additional_params': mark_safe(
+                '&'.join('export_id=%(export_id)s' % {
```

Comment:
true, though looks like there's no defined standard for it: http://stackoverflow.com/questions/24059773/correct-way-to-pass-multiple-values-for-same-parameter-name-in-get-request